docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md

```diff
@@ -0,0 +1,256 @@
+    | **Container Disk Upload Method**            | Indicate whether to upload the image directly to the target cluster as a `DataVolume` or build and push a Docker image. You will need to provide an existing image registry if you select Docker. | `DataVolume` / `Docker Image`     |
+    | **Kubeconfig Path**                         | The path to the kubeconfig file you have uploaded to your VM.                                                                                                                                     |                                   |
+    | **DataVolume Namespace**                    | The namespace to create your `DataVolume`, if you selected this option previously.                                                                                                                |                                   |
+    | **DataVolume Name**                         | The name of your `DataVolume`.                                                                                                                                                                    |                                   |
+    | **Overhead Percentage for DataVolume Size** | Set an overhead percentage for your `DataVolume` compared to the OVA specification. This parameter is optional and can be skipped with the value `-1`.                                            |                                   |
+    | **Access Mode for the PVC**                 | Set the access mode for your `DataVolume`. Ensure that your configured CSI supports your selection.                                                                                               | `ReadWriteMany` / `ReadWriteOnce` |
+    | **Create a PVC with VolumeMode=Block**      | Indicate whether to set `Block` volume mode on the `DataVolume`.                                                                                                                                  | `y` / `N`                         |
+    | **StorageClass**                            | The storage class on the destination that will be used to create the VM volume.                                                                                                                   |                                   |
+    | **CDI upload proxy URL**                    | Optionally, provide a URL to upload the CDI custom resource.                                                                                                                                      |                                   |
+
+11. The import may take a few minutes to complete. The Palette CLI outputs the path for your OVA configuration file.
+    Make a note of it.
+
+12. Your imported OVA is now ready to be deployed to a new VM inside your VMO cluster. Execute the following command to
```

Comment:
We may want to include the following warning:
> The DataVolume upload will fail if you are in a proxied environment and have not properly configured the CDI custom resource. Refer to the [CDI configuration options](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/cdi-config.md#options) as required.
>
> If you have configured NGINX ingress + MetalLB and you can access the NGINX load balancer IP, but not the CDI upload proxy IP, add an entry to `/etc/hosts` that maps the CDI upload proxy host name, e.g. `cdi-uploadproxy.mycompany.io`, to the NGINX load balancer’s public IP.

Reply:
And we should definitely add this info box:

> If the DataVolume upload fails for any reason, re-run the upload after debugging any CDI issues using: `palette vmo import-ova -s -f /path-to-ova-config.yaml --skip-convert`

Reply:
Added thank you so much!